### PR TITLE
32 test farm factory

### PR DESCRIPTION
**Pull request by BritikovKI**
I've achieved 100% code coverage for farm factory.

### CODE REVIEW

File: contracts/schemes/FarmFactory.sol [hunk 1]
```diff
@@ -172,13 +180,11 @@ contract FarmFactory {
 			address(_farm),
 			abi.encodeWithSelector(
 				_farm.notifyRewardAmount.selector,
-				_amount
+				_amount + oldBalance
```
Comment (kinspotting) [inline contracts/schemes/FarmFactory.sol:183]: pls use SafeMath for mathematical operations

**Review by kinspotting — COMMENTED** [review]
needs minor changes

File: contracts/schemes/FarmFactory.sol (hunk 1 above)
Comment (BritikovKI) [reply contracts/schemes/FarmFactory.sol]: It is add now

**Review by kinspotting — APPROVED** [review]
lgtm

**Pull request merged**